docs/getting_started.rst

```diff
@@ -1,24 +1,13 @@
-Getting Started
-===============
+.. _aiapy-getting-started-index:
 
-The stable version of ``aiapy`` is available through both PyPI via ``pip``,
+******************************
+Getting started with ``aiapy``
+******************************
 
-.. code-block:: shell
+``aiapy`` is a community-developed, free and open-source library to to convert AIA images from level 1 to level 1.5, compute point spread function deconvolution, and computing the wavelength response functions for the EUV channels.
```

Comment:
```suggestion
``aiapy`` is a community-developed, free and open-source Python library for analyzing data from the Atmospheric Imaging Assembly (AIA) onboard the Solar Dynamics Observatory (SDO). 
Capabilities include converting AIA images from level 1 to level 1.5, deconvolving images with the instrument point spread function, and computing the wavelength response functions for the EUV channels.
```

Reply:
Actually, maybe this text can just be removed altogether. There's not really a need to repeat what's on the main page.

Reply:
I can remove it.